packages/babel-plugin-transform-async-to-promises/src/index.js

```diff
@@ -0,0 +1,112 @@
+/* Babel 7 transform */
+
+const babylon = require("babylon");
+const generate = require("@babel/generator").default;
+
+function printNode(ast) {
+  return generate(ast, {}).code;
+}
+
+function parse(code) {
+  return babylon.parse(code, {
+    allowImportExportEverywhere: true,
+    allowReturnOutsideFunction: true,
+    allowSuperOutsideMethod: true,
+  }).program;
+}
+
+const transform = require("nodent-transform").transform;
+
+function transformAsyncToPromises(api, options) {
+  let requiresTranspilation;
+  const runtime = options.runtime ? options.runtime : null;
+  let opts = {
+          // Code generation options
+          es6target: false,
+          babelTree: true,
+          engine: false,
+          generators: false,
+          promises: true,
+          lazyThenables: false,
+          wrapAwait: true,
+          noRuntime: !runtime,
+          $runtime: runtime,
+          generatedSymbolPrefix: "$",
+          $return: "$return",
+          $error: "$error",
+          $arguments: "$args",
+          $Promise: "Promise",
+          $asyncspawn: "$asyncspawn",
+          $asyncbind: "$asyncbind",
+          $makeThenable: "$makeThenable",
+        }; 
+
+  if (options.codeGenerationOptions) {
+      let keys = Object.keys(options.codeGenerationOptions);
+      for (let i=0; i<keys.length;i++) {
+	  if (keys[i] in opts)
```

Comment:
This line seems to be indented weirdly?

Reply:
No idea why - I'd guess something put tabs in place of spaces or vice versa. I'm not able to fix this at the moment (away from my home setup with the correct tools). Feel free to edit to match your preferred style.

Reply:
...I just had a go with the github editor, but TBH I'm not familiar with how these are best merged online.

Reply:
So you're ok to accept the PR?

Reply:
Oops, sorry, I'll dismiss my review. I'm not personally confident in approving this, but I definitely am not blocking it further :-)